backend/src/models/bike.model.ts

```diff
@@ -5,11 +5,12 @@ enum BikeStatus {
 
```

Comment:
review optional properties on interfaces

Reply:
Did it. Thanks for pointing.